plugins/woocommerce-admin/client/payments/payment-recommendations-wrapper.tsx

```diff
@@ -21,15 +21,11 @@ export const PaymentRecommendations: React.FC< EmbeddedBodyProps > = ( {
 	tab,
 	section,
 } ) => {
-	if ( page === 'wc-settings' && tab === 'checkout' && ! section ) {
-		if (
-			window?.wcAdminFeatures?.[
-				'reactify-classic-payments-settings'
-			] === true
-		) {
-			return null;
-		}
-
+	if (
+		page === 'wc-settings' &&
+		tab === 'checkout' &&
+		( ! section || section === 'main' )
```

Comment:
I added `section === main` since it's possible to render the main payments screen by adding `&section=main` parameter in URL